### PR TITLE
database performance audit

### PR DESCRIPTION
**Pull request by XsidX**
For this PR, We:
  - [add email column on owners table](https://github.com/XsidX/vet-clinic-db/commit/477c742b61745188385c020963ec55b06a80d147)
  - [insert 3M+ records into the visits & owners tables](https://github.com/XsidX/vet-clinic-db/commit/a437c4d1c169642e740f18d2d717aed96d2e3ab2)
  - [add query analylisis](https://github.com/XsidX/vet-clinic-db/commit/c9914b79d6cd748995543b74425f54bb8d699fee)
  - [add performance audit screenshots](https://github.com/XsidX/vet-clinic-db/commit/5f0760e7bec70e51421d356727f36435643780ea)
  - @ChrispaixK
[move create index to the schema file](https://github.com/XsidX/vet-clinic-db/pull/5/commits/4c8eb1eb993faa6aec663bf89c1b4e26f219d1a0)

**Screenshots**

> **visits where animal_id**
visits-where-animal_id

> **visits where animal_id after adding an index**
with-index-visits-where-animal_id

> **visits where vet_id**
visits-where-vet_id

>  **visits where vet_id after adding an index**
with-index-visits-where-vet_id

> **owners where email_id**
owners-where-email

> **owners where email_id after adding an index**
with-index-owners-where-email

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -213,8 +213,31 @@ ORDER BY COUNT(species.name) DESC
 LIMIT 1;
 
 
+------------------------------------------------------------------------------------------------------------------------------
+  -- performance audit
+------------------------------------------------------------------------------------------------------------------------------
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4
+-- the above query returns an execution time of 2757.049 ms
+CREATE INDEX visits_animal_id_idx ON visits (animal_id);
```
Comment (danmainah) [inline queries.sql]: - [ ] @ team great work on your queries, however, the indexes creation should be done in the `schema.sql` file, kindy move them to your schema file, your queries file should contain only queries to your database.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @XsidX and @ChrispaixK , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (XsidX) [reply queries.sql]: Thanks for the feedback. On it!

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @XsidX  🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**